app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
It looks like the app is having issues with password confirmation when you try to create a new user. I tried creating a new user without the `:confirmable` and it worked fine. However, with `:confirmable`, it does not work but throws an error as shown below :point_down: 
Kindly fix it and make sure it works well with no errors. 

![Screenshot from 2023-06-21 23-22-36](https://github.com/jaimevillegas/rails-blog-app/assets/65233860/d6e3cbbe-f4bc-4502-bc02-2012f415a8e2)